src/main/fc/rc.c

```diff
@@ -682,9 +690,15 @@ FAST_CODE void processRcCommand(void)
 
 #ifdef USE_INTERPOLATED_SP
     if (isRxDataNew) {
+        float rcCommandf;
```

Comment:
Not sure why you are pulling this out to here - the code will be easier to read if you only declrare it before the `if (i == FD_YAW) {` line.

Reply:
Done